### ISSUE TITLE
"os" package not functioning correctly

### ISSUE THREAD

**Issue by 0pcom** [skycoin/cx#411]
**Describe the bug**

When the following code is run, nothing is printed. 
It doesn't like anything inside the parentheses either.

**To Reproduce**

```
package main
import "os"

func main () {
  var wd str
  wd = os.GetWorkingDirectory()
str.print(wd)
}
```

**Expected behavior**
I assume it should be putting _something_ into `var wd` and print it back to the screen.

**System:**
 - OS: Archlinux
 - 0.7beta

**Additional Context**

The other os functions don't seem to work correctly either;

```
os.Open()
os.Close()
```

else their documentation is missing something.

**Comment by ingwal** [comment]
Yes, the os package is indeed not good.  The Open() and Close() functions do work, but the problem is that Open() only creates an open file, it doesn't return a handle to it.  Internally you can access it, and this is what CXFX does when it reads patterns and other stuff.  But obviously, CX must have a working file I/O package.  We will redesign this for CX 0.8, but we expect something experimental already for CX 0.7.2.  It's not so easy to get it right, but at least we will create something so that file handling becomes possible.
